### PR TITLE
Index cluster_uuid in elasticsearch/node metricset

### PR DESCRIPTION
**Pull request by ycombinator**
This PR teaches the `elasticsearch/node` metricset to index the Elasticsearch `cluster_uuid` as the module-level `cluster.id` field.

### CODE REVIEW

**Comment by ruflin** [comment]
There still seems to be some troubles with CI

**Comment by ycombinator** [comment]
@ruflin I addressed the CI troubles. Ready for your review again. Thanks!

File: metricbeat/module/elasticsearch/node/node_test.go [hunk 1]
```diff
@@ -48,7 +48,6 @@ func TestFetch(t *testing.T) {
 				w.WriteHeader(200)
 				w.Header().Set("Content-Type", "application/json;")
 				w.Write([]byte(response))
-				assert.Equal(t, "/_nodes/_local", r.RequestURI)
```
Comment (ruflin) [inline metricbeat/module/elasticsearch/node/node_test.go]: Is this not correct anymore?

Comment (ruflin) [reply metricbeat/module/elasticsearch/node/node_test.go]: That it works locally and fails on CI makes me worry now because I think it passed previously.

Comment (ycombinator) [reply metricbeat/module/elasticsearch/node/node_test.go]: Alright, digging into it some more...

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
@ruflin All the things are green now on this PR and review feedback has been addressed. Please take another gander whenever you get a chance. Thanks!

**Pull request merged**